app/component/toast.tsx

```diff
@@ -0,0 +1,33 @@
+import Failed from "react:~assets/failed.svg"
+import Submit from "react:~assets/submit.svg"
+import Success from "react:~assets/success.svg"
+
+export type ToastStatus = "success" | "failed" | "submit"
+
+type ToastProps = {
+  status: ToastStatus
+  message: string
+}
+
+const statusIcon: Record<ToastStatus, JSX.Element> = {
+  success: <Success />,
+  failed: <Failed />,
+  submit: <Submit />
+}
+
+const statusTextColor: Record<ToastStatus, string> = {
+  success: "text-[#7FFF9C]",
+  failed: "text-[#FF9393]",
+  submit: "text-[#7EE0FF]"
+}
+export const Toast = (props: ToastProps) => {
+  const { status, message } = props
+  return (
+    <div className="absolute w-[358px] top-[32px] flex gap-[8px] items-center bg-black rounded-2xl p-[16px] shadow-[4px_8px_0px_0px_rgba(0,0,0,0.16)]">
```

Comment:
I am thinking that `static` might be more proper than `absolute`? What do you think? 🤔

Reply:
I used `absolute` because `<Toast/>` should hover on top of the screen.

| without toast hover | with toast hover |
| ------ | ----------- |
| <img width="160" alt="截圖 2024-07-11 下午4 51 06" src="https://github.com/pilagod/waallet-extension/assets/61860963/cdcb5966-0042-4809-94e7-27d00d294920"> | <img width="256" alt="截圖 2024-07-11 下午4 50 51" src="https://github.com/pilagod/waallet-extension/assets/61860963/e36cfa49-d7fc-4b11-8348-05479acaeff6"> |

Reply:
In the video you manually scroll to top in order to see the toast, if this is the case, user might miss the message when pinning it at absolute top. (Not sure if I understand it correctly 🙈)

Reply:
I see! I change the screen height: [modify global height to 600px](https://github.com/pilagod/waallet-extension/pull/158/commits/3732c8c501797c277044c426f34a0211bfb0a8d1)
Now it will automatically go to the top and you can see the toast directly.


https://github.com/user-attachments/assets/f8f087d7-b80d-4c1e-a731-2e7e15d5ecab

Reply:
Nice, LGTM 🚀